src/Features/Lsif/Generator/Generator.cs

```diff
@@ -33,6 +34,20 @@ internal sealed class Generator
         private const bool FoldingRangeProvider = true;
         private const bool DiagnosticProvider = false;
 
+        private static readonly LspProtocol.ClientCapabilities LspClientCapabilities = new()
```

Comment:
So here's a general question for LSIF: do we need a switch to the generator for things like this?

Reply:
(this question isn't blocking, but just something to think about)

Reply:
Great question, I think eventually ya. IMO it shouldn't have too many knobs, maybe just  a "what LSP version do you holistically want me to generate for"?